src/rpc/blockchain.cpp

```diff
@@ -331,18 +331,18 @@ UniValue getdifficulty(const UniValue& params, bool fHelp)
 
 std::string EntryDescriptionString()
 {
-    return "    \"size\" : n,             (numeric) transaction size in bytes\n"
+    return "    \"size\" : n,             (numeric) virtual transaction size. Differs from size for witness transactions since witness size is discounted. BIP 141.\n"
```

Comment:
How about "virtual transaction size. This is different from actual serialized size for witness transactions as witness data is discounted (BIP 141)"